index.js

```diff
@@ -29,12 +29,10 @@ module.exports = function (app, translate) {
     }
 
     if (!params) {
-      params = this.request.body
```

Comment:
maybe we should detect the method and only use `this.query` when request method is `GET`?I don't want to mixed `query` and `body` here.